Production_scripts/convert_gp_costs.R

```diff
@@ -0,0 +1,143 @@
+#####################################################
+# Costs - GP Out of Hours
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description -
+#####################################################
+
+
+library(dplyr)
+library(ggplot2)
```

Comment:
Add required packages (or make calls to functions explicit with `::`)

```suggestion
library(dplyr)
library(ggplot2)
library(createslf)
library(tidyr)
library(purrr)
```